core/server/master/src/main/java/alluxio/master/AlluxioMasterProcess.java

```diff
@@ -196,6 +201,31 @@ public boolean isServing() {
     return mIsServing;
   }
 
+  @Override
+  public List<ConfigInfo> getConfigInfoList() {
+    List<ConfigInfo> configInfoList = new ArrayList<>();
+    String alluxioConfPrefix = "alluxio";
+    Set<String> alluxioConfExcludes = Sets.newHashSet(
+        PropertyKey.MASTER_WHITELIST.toString());
+    for (Map.Entry<String, String> entry : Configuration.toMap().entrySet()) {
```

Comment:
can iterate over keyset if we are only using the keys

Reply:
We also use the values now, thanks for pointing out that!